deps/build.jl

```diff
@@ -70,4 +70,8 @@ provides(BuildProcess, (@build_steps begin
     end
 end), [libfftw, libfftwf])
 
-BinDeps.@install Dict([:libfftw => :libfftw, :libfftwf => :libfftwf])
+if is_windows()
+    BinDeps.@install Dict([:libfftw => :libfftw, :libfftwf => :libfftwf])
+else
+    BinDeps.@install Dict([:libfftw3_threads => :libfftw, :libfftw3f_threads => :libfftwf])
```

Comment:
how about this instead:
```julia
BinDeps.@install Dict(Symbol(libfftw_name) => :libfftw, Symbol(libfftwf_name) => :libfftwf)
```
that way you avoid the if statement and also just reuse the const variables so that if they change these will be updated